spec/extensions/extension_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+describe Blueprinter::Extension do
+  subject { Class.new(described_class) }
+
+  context 'hooks' do
+    let(:blueprint) { Class.new(Blueprinter::V2::Base) }
+    let(:field) { Blueprinter::V2::Field.new(name: :foo, from: :foo) }
+    let(:object) { { foo: 'Foo' } }
+    let(:context) { Blueprinter::V2::Context }
+
+    it 'should default collections? to false' do
```

Comment:
Super *nit*, but I'd suggest omitting the "shoulds" in `it` assertions (e.g. use `it 'defaults collections? to false'`).